lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -38,6 +38,19 @@ abstract class EcdhPrivateKey {
     return impl.ecdhPrivateKey_generateKey(curve);
   }
 
+  /// Derives a shared secret from two ECDH key pairs using the private key from one pair 
+  /// and the public key from another.
+  /// 
+  /// The shared secret is identical whether using A's private key and B's public key, 
+  /// or B's private key and A's public key, enabling secure key exchange between the 
+  /// two parties.
+  ///
+  /// [length] specifies the length of the derived secret in bits.
```

Comment:
Should we make a list bullet point list with what lengths the different curves can provide:
```md
[length] specifies the length of the derived secret in bits. The maximum length depends on the [EllipticCurve] used in the keys:
...
 * [EllipticCurve.p521] can provide up to 528 bits.
```

or something like that.

Reply:
We'd have to find out how much each curve can derive, and make sure we have tests :D

Reply:
Actually, let's do this in a follow up issue.. That way we land now.

Landing now is always better :rofl:

Reply:
Filed https://github.com/google/webcrypto.dart/issues/130